docs/pages/yaml-reference.md

```diff
@@ -97,6 +93,20 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
   * `strategy`
 
     This field isn't used! TODO(nickbp) remove.
+   
+  * `networks`
+
+    Allows the pod to join any number of virtual networks on the DC/OS cluster, however the only supported virtual network at present is the `dcos` overlay network. To have the pod join the overlay add the following:
```

Comment:
Thing is that overlay networks are a subset of virtual networks (to the best of my understanding). I don't know if you can join any number of _overlay_ networks...